src/logging.c

```diff
@@ -231,3 +231,30 @@ void log_walls_detection(void)
 		 "\"wall_front\":%d}",
 		 left_wall, right_wall, front_wall);
 }
+
+/**
+ * @brief Log gyroscope's Z-axis raw readings from systick, published for
+ * real-time.
```

Comment:
Try to stick to 1-line `@brief`s. (here and in other functions)

You could say maybe "Publish gyroscope's Z-axis raw readings in real-time."? Or even without the "real-time".

Reply:
Ok, updated every 2-line brief.